src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -39,27 +39,13 @@ public void setUp() {
     @Test
     void createStation() {
         // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        ValidatableResponse response = create_station("강남역");
```

Comment:
공통으로 사용할 수 있는 기능을 메서드로 잘 추출해주셨네요 :)

테스트에서 한글명 메서드를 활용하면 조금 더 의미전달이 쉬운 테스트코드를 구성할수도 있습니다 ㅎㅎ
테스트의 가독성도 매우 중요한 요소이고, 인수테스트는 더더욱 의미전달이 쉬우면 더 좋지 않을까 해서 코멘트 드려요 😁

```suggestion
        ValidatableResponse response = 지하철역_생성("강남역");
```
예시 일 뿐이고, 어떤 형태든 좋습니다 :)

Reply:
넵넵 한글이 이해하기 더 좋은 것 같네요..!